### PR TITLE
Added Balances Pallet

### PR DESCRIPTION
**Pull request by SouravMishra3256**

### CODE REVIEW

File: balances/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+[dependencies.frame-benchmarking]
+default-features = false
+git = 'https://github.com/paritytech/substrate.git'
+optional = true
+branch = 'polkadot-v0.9.26'
+
+[dependencies.frame-support]
+default-features = false
+git = 'https://gitThishub.com/paritytech/substrate.git'
```
Comment (fennsaji) [inline balances/Cargo.toml:35]: gitThishub?

Comment (SouravMishra3256) [reply balances/Cargo.toml]: Changed It To The Correct One

File: balances/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+[package]
+name = "balances"
```
Comment (fennsaji) [inline balances/Cargo.toml:2]: Make it pallet-balances

Comment (SouravMishra3256) [reply balances/Cargo.toml:2]: Done

File: balances/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,2176 @@
...
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[macro_use]
+mod tests;
+mod benchmarking;
+mod tests_composite;
+mod tests_local;
+#[cfg(test)]
+mod tests_reentrancy;
```
Comment (fennsaji) [inline balances/src/lib.rs:162]: I don't see any file with this name

Comment (SouravMishra3256) [reply balances/src/lib.rs:162]: Fixed

**Pull request closed without merging**